polaris-migrator/README.md

```diff
@@ -235,11 +235,18 @@ Be aware that this may also create additional code changes in your codebase, we
 npx @shopify/polaris-migrator replace-sass-spacing <path>
 ```
 
-## Creating a migration
+## Creating Migrations
```

Comment:
The entry point is still the same, it's just executed inside the `createSassMigration` function instead of copy+pasta'ing it into every migration file.

It looks like that chunk of code is very unlikely to change, so would be better moved out of the individual migrations IMO.

Reply:
I see, makes sense! I was trying to keep it relatively "generic" in saying that there is a single entry point then you can do whatever file/AST modifications you'd like from there.

Can we save these edits for a separate PR? I'd like to hash on them a bit and add a section for TS migrations before making these updates

Reply:
Yeah, no worries: I've pulled them out so we can discuss separately to this work 👍